app/models/TraderProfile.scala

```diff
@@ -34,6 +35,7 @@ object TraderProfile {
 
   def build(answers: UserAnswers, eori: String): EitherNec[ValidationError, TraderProfile] =
     (
+      Right(eori),
```

Comment:
We actually don't need the EORI in the body, just in the path.

https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/trader-goods-profiles/1.0/oas/page#tag/Traders-profile/operation/updateTraderGoodsProfile

Reply:
So this is to make parsing the body we get back easier, as the body we get back has the actor id and the eori in it

Reply:
changed it and added the data store profile into test